uw-research-computing/gpu-jobs.md

```diff
@@ -337,11 +337,11 @@ server, including:
 		NVIDIA website</a>.</td>
 	</tr>
 	<tr>
-		<td><code>GPUs_DriverVersion</code></td>
+		<td><code>GPUs_</code>)<code>DriverVersion</code></td>
 		<td><b>Not</b> the version of CUDA on the server, but the maximum CUDA runtime version supported by the GPU drivers on the server. </td>
```

Comment:
```suggestion
		<td><b>Not</b> the version of CUDA on the server or the NVIDIA driver version, but the maximum CUDA runtime version supported by the NVIDIA driver on the server. </td>
```
This is unrelated to your change, but I find there is still a lot of confusion about this attribute.